tests/conftest.py

```diff
@@ -86,3 +86,4 @@ def one_task_belongs_to_one_goal(app, one_goal, one_task):
     goal = Goal.query.first()
```

Comment:
You mentioned in Learn that you wrote pytests, but I don't see them. Please point them out to me if you'd like feedback on them!